guides/common/modules/proc_performing-discovery-in-pxeless-mode.adoc

```diff
@@ -0,0 +1,57 @@
+[id="performing-discovery-in-pxeless-mode_{context}"]
+= Performing Discovery in PXE-less mode
+[[Implementing_PXE_less_Discovery_{context}]]
+
+Discovery in PXE-less mode uses the Discovery ISO and requires you to attend to the process.
+
+.Prerequisites
+ifdef::satellite[]
+* You have installed the Foreman Discovery image.
+For more information, see xref:Installing_the_Discovery_Service_{context}[].
+endif::[]
+ifndef::satellite[]
+* You have installed the Foreman Discovery image or built a custom Discovery image ISO.
+For more information, see one of the following resources:
+** xref:Installing_the_Discovery_Service_{context}[]
+** xref:Building_a_Discovery_Image_{context}[]
+endif::[]
+
+.Procedure
+. Copy the Discovery ISO to either a CD, DVD, or a USB stick.
+For example, to copy to a USB stick at `/dev/sdb`:
++
+[options="nowrap" subs="+quotes"]
+----
+# dd bs=4M \
+if=/usr/share/foreman-discovery-image/foreman-discovery-image-_version_.iso \
+of=/dev/sdb
+----
+. Insert the Discovery boot media into a bare-metal host, start the host, and boot from the media.
+The Discovery Image displays an option for either *Manual network setup* or *Discovery with DHCP*:
++
+
+* If you select *Manual network setup*, the Discovery image requests a set of network options.
+This includes the primary network interface that connects to {ProjectServer}.
+This Discovery image also asks for network interface configuration options, such as an *IPv4 Address*, *IPv4 Gateway*, and an *IPv4 DNS* server.
++
+After entering these details, select *Next*.
+* If you select *Discovery with DHCP*, the Discovery image requests only the primary network interface that connects to {ProjectServer}.
+It attempts to automatically configure the network interface using a DHCP server, such as one that a {SmartProxyServer} provides.
+. After the primary interface configuration, the Discovery image requests the *Server URL*, which is the URL of {ProjectServer} or {SmartProxyServer} offering the Discovery service.
+For example, to use the integrated {SmartProxy} on {ProjectServer}, use the following URL:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+https://{foreman-example-com}:{smartproxy_port}
+----
+. Set the *Connection type* to `Proxy`, then select *Next*.
```

Comment:
```suggestion
. Enter `Proxy` as the connection type and then click *Next*.
```
UI field names are used as adjectives, not nouns. If "Next" is a button, the correct verb is "click".